### PR TITLE
add balenacloud

### PR DESCRIPTION
**Pull request by TheOrangeCat**

### CODE REVIEW

File: balenaengine/Pkgfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Description: An engine purpose-built for embedded and IoT use cases, based on Moby Project technology from Docker
+# URL:         https://www.balena.io/engine/
+# Maintainer:  TheOrangeCat, vdos63@gmail.com
+
+name=balenaengine
+version=18.9.5
+source=("https://github.com/balena-os/balena-engine/archive/v$version.tar.gz")
+BOOTSTRAP=no
```
Comment (protonesso) [inline balenaengine/Pkgfile]: you can remove BOOTSTRAP=no, not needed.

Comment (TheOrangeCat) [reply balenaengine/Pkgfile]: k bye

Comment (protonesso) [reply balenaengine/Pkgfile]: also replace vdos63@gmail.com to vdos63 at gmail dot com

File: balenaengine/Pkgfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+build() {    
+	curl -sfL https://balena.io/engine/install.sh | sh
```
Comment (protonesso) [inline balenaengine/Pkgfile]:
```
build() {
	case $BARCH in
		x86_64)
			export BALENAARCH="x86_64"
			;;
		i586)
			export BALENAARCH="i386"
			;;
		aarch64)
			export BALENAARCH="aarch64"
			;;
		armv7l)
			export BALENAARCH="armv7"
			;;
		armv6l)
			export BALENAARCH="armv6"
			;;
		*)
			echo "Architecture is not set or is not supported by Balena Engine"
			exit 1
esac
cd "$SRC"
mkdir -p "$PKG"/usr/bin
wget https://github.com/resin-os/balena-engine/releases/download/v$version/balena-engine-v$version-$BALENAARCH.tar.gz
tar -xvf balena-engine-v$version-$BALENAARCH.tar.gz -C "$PKG"/usr/bin
}

Comment (protonesso) [reply balenaengine/Pkgfile]: also set version to 17.12.0 and remove "source" variable

Comment (TheOrangeCat) [reply balenaengine/Pkgfile]: ok

Comment (TheOrangeCat) [reply balenaengine/Pkgfile]: why not curl

Comment (TheOrangeCat) [inline balenaengine/Pkgfile]:
```suggestion
	cat <<EOF


   Installation successful!
 _           _
| |__   __ _| | ___ _ __   __ _
| '_ \\ / _\` | |/ _ \\ '_ \ / _\` |
| |_) | (_| | |  __/ | | | (_| |
|_.__/ \__,_|_|\___|_| |_|\__,_|

the container engine for the IoT
EOF

```

Comment (protonesso) [reply balenaengine/Pkgfile]: if you want to use curl please add `# Depends on: curl~ and use curl with those opts
```
-L --fail --ftp-pasv --retry 3 --retry-delay 3 -O $PKGMK_CURL_OPTS
```
to make pkgmk users happy :)

Comment (protonesso) [reply balenaengine/Pkgfile]: nice

**Pull request merged**